.editorconfig

```diff
@@ -5,3 +5,9 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+
+[/Symfony/CS/Tests/Fixtures/**/*]
+end_of_line = ignore
+insert_final_newline = ignore
+indent_style = ignore
+trim_trailing_whitespace = ignore
```

Comment:
What I meant is to create an `.editorconfig` with the following content

```yml
root = true
```

in [`/Symfony/CS/Tests/Fixtures`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/1.13/Symfony/CS/Tests/Fixtures) - maybe that helps?